Mod2/Lessons/Week5/ManyToManyWithAnORM.md

```diff
@@ -0,0 +1,64 @@
+<br>
+
+❓Why might the teams building frameworks use versions instead of just releasing new features as they build them?
+
+<details><summary>Some of the many reasons..</summary>
+
+- Sometimes an update creates what's called a `breaking change` where with the new version some old code won't work anymore. You don't want your software to break without warning!
+- It's easier for users to see what's new by viewing `release notes`.
+- Developers can release `beta versions` for early adopters to try out the latest features and find bugs. Then they can later make that version a `stable version` that is a well-tested version most users will prefer.
+</details>
+
+<br>
+
+When the development team releases a new version of their software, they also create release notes. Let's take a look at the release notes for EF Core 5.0: https://learn.microsoft.com/en-us/ef/core/what-is-new/ef-core-5.0/whatsnew
+
+
+Something to be aware of when you're looking up tutorials or blog posts is that sometimes you will find tutorials that have not been updated and are still referencing an old version. For example, this tutorial is still teaching the old way of building a many-to-many relationship in EF. https://www.entityframeworktutorial.net/efcore/configure-many-to-many-relationship-in-ef-core.aspx
+
+<!-- Instructor note, I recommend giving a quick note about the gradual release of responsibility here and why we are having students learn on their own. -->
+
+### Learning From the Documentation
+
+Your goal is to create a new table to keep track of `employees` and implement a many-to-many relationship between `employees` and `events`. This should be a many-to-many relationship because each event can be hosted by many employees and each employee can host many events.
+
+As part of the gradual release of responsibility, you all are ready to learn how to build this from the documentation.
+
+Use this documentation to learn how to implement this data relationship using Entity Framework: https://learn.microsoft.com/en-us/ef/core/modeling/relationships?tabs=fluent-api%2Cfluent-api-simple-key%2Csimple-key#many-to-many
+
+After you have created a many-to-many relationship use PGAdmin to make an ERD for your database and verify your tables look as expected. You should see your join table! There is something subtly different with the naming convention from what we have done in the past, do you see the difference?
+
+<!-- Instructor note, in the autogenerated table the column names are pluralized like the table names. So the column is called rooms_id instead of room_id -->
+
+<!-- Instructor note, as you are jumping into the breakout rooms. As someone who successfully implemented the many-to-many to share their screen and walk through their process. -->
+
+
+<!-- Open question, should we be doing things with CRUD here and how that works with a many to many relationship? Not really going to know until we build the next unit. I think leave it unless later we realize we want it. -->
```

Comment:
I would include something here on how to create a record that has a many to many relationship, but I don't think we need to go into detail about the RUD portion :)